src/css/Calculator.css

```diff
@@ -0,0 +1,63 @@
+.calculator {
+  display: flex;
+  flex-direction: column;
+  width: 40%;
+  background-color: #333;
+  border-radius: 5px;
+  padding: 10px;
+}
+
```

Comment:
- Great Job for having this beautiful UI, I think for this milestone you have to stick to the given design to fulfill the [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md) 👍 
#### Your design 👇
![Screenshot 2023-07-05 152409](https://github.com/Braingenius2/Math-Magicians2/assets/52010789/e8e24e91-cc5a-4809-8b40-88ab94af9ffc)
#### Given Design 👇
![Screenshot 2023-07-05 152504](https://github.com/Braingenius2/Math-Magicians2/assets/52010789/93d4c8bd-f098-4456-8972-cddfc4d76015)